src/frontend/packages/store/src/public-api.ts

```diff
@@ -6,4 +6,20 @@
  export * from './helpers/store-helpers';
 
  // App State
- export { AppState } from './app-state';
\ No newline at end of file
+ export { AppState } from './app-state';
+
+ // Used by store testing module
+ export { CATALOGUE_ENTITIES, EntityCatalogFeatureModule } from './entity-catalog.module';
+ export { entityCatalog, TestEntityCatalog } from './entity-catalog/entity-catalog';
+ export { EntityCatalogEntityConfig } from './entity-catalog/entity-catalog.types';
+ export { endpointEntityType, stratosEntityFactory } from './helpers/stratos-entity-factory';
+ export { appReducers } from './reducers.module';
```

Comment:
The reducers are pretty private stuff. I can understand why they're used in tests but we're lumping them in here with production code. It's not a blocker, but if we find we expose a lot of internal stuff then we might need test specific exporters.